asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePage.java

```diff
@@ -232,7 +239,8 @@ private static Options options(Path templatesFolder, Path baseFolder, Path gener
 
             try {
                 String referencedPageContent = readIntoString(new FileInputStream(referencedPagePath.toFile()), sourceEncoding);
-                String referencedPageTitle = pageTitle(referencedPageContent, userAttributes, pageTitlePostProcessor);
+                Document referencedDocument = ASCIIDOCTOR.load(referencedPageContent, Options.builder().parseHeaderOnly(true).build());
```

Comment:
setting `parseHeaderOnly` to parse only the header, since we need to get only title of the page here